js-control-structures/js-grades-excercise/js-grades-excercise.js

```diff
@@ -0,0 +1,32 @@
+const examPointsInput = prompt('Please add your points here');
+let grade;
+
+if (examPointsInput>=0 && examPointsInput<=100 && !examPointsInput.split('.')){
+    if(examPointsInput>=51 && examPointsInput<=60){
+        grade=6;
+        console.log(`Congratulations, you have passed the exam. Your grade is: ${grade}`);
```

Comment:
You have duplicated the console log in each IF block. You can move outside of it on the bottom and print the message from there. This is possible because you have the grade variable and the value will be available for you. 

The place where you can log this message is below line: 28

Reply:
Each if else block can just assign the grade value from 5-10.